src/it/resources/com/google/checkstyle/test/chapter5naming/rule528typevariablenames/InputMethodTypeParameterName.java

```diff
@@ -43,9 +43,9 @@ <E extends T> void getMore() {
     };
 
     Other2 o =
-            new Other2() {
-              <EE> void getMoreFoo() { // violation 'Method type name 'EE' must match pattern'
-              }
-            };
+        new Other2() {
+          <EE> void getMoreFoo() { // violation 'Method type name 'EE' must match pattern'
+          }
+        };
```

Comment:
formatter changed indentation, previously I might have used google-java-format plugin in intellij for formatting but this time I used vs-code's google-java-format plugin. There might be little difference in both of these plugins so the code didn't got formatted properly previously.